docker-compose.yaml

```diff
@@ -16,7 +16,7 @@ version: '3.5'
 
 services:
   tcf:
-    container_name: tcf
+    container_name: tcf-manju
```

Comment:
I suggest restoring the name `tcf` or using another name than` tcf-manju`. Was this for debugging?

Reply:
this file was mistakenly added in the commit list. I will remove it.